bundles/org.openhab.binding.zwavejs/src/main/java/org/openhab/binding/zwavejs/internal/action/ZwaveJSActions.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.zwavejs.internal.action;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.zwavejs.internal.BindingConstants;
+import org.openhab.binding.zwavejs.internal.handler.ZwaveJSBridgeHandler;
+import org.openhab.core.automation.annotation.ActionInput;
+import org.openhab.core.automation.annotation.RuleAction;
+import org.openhab.core.common.ThreadPoolManager;
+import org.openhab.core.thing.binding.ThingActions;
+import org.openhab.core.thing.binding.ThingActionsScope;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ServiceScope;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author Leo Siepel - Initial contribution
+ */
+@Component(scope = ServiceScope.PROTOTYPE, service = ZwaveJSActions.class)
+@ThingActionsScope(name = "zwavejs")
+@NonNullByDefault
+public class ZwaveJSActions implements ThingActions {
+    private final Logger logger = LoggerFactory.getLogger(ZwaveJSActions.class);
+    private static final ScheduledExecutorService SCHEDULER = ThreadPoolManager
+            .getScheduledPool(BindingConstants.BINDING_ID);
+    private static final int INCLUSION_EXCLUSION_TIMEOUT_SECONDS = 30;
+    private @Nullable ZwaveJSBridgeHandler handler;
+
+    @RuleAction(label = "start inclusion", description = "Put the controller for 30s in network wide inclusion mode")
```

Comment:
Please consider i18n translation.
Same comment for other actions.

The "convention" for action description is to use: "Put**s** the ...". It is fine for label.
To be applied to other actions.

Reply:
Fixed